core/shared/src/main/scala/scalaz/zio/Fiber.scala

```diff
@@ -21,7 +21,7 @@ package scalaz.zio
  * } yield a
  * }}}
  */
-trait Fiber[+E, +A] { self =>
+trait Fiber[+E, +A] extends Serializable { self =>
```

Comment:
This is one I would not change, because a Fiber represents a running computation. It's possible to make it serializable but only with additional development.